cfht/ref_cats/sdss-dr9-fink-v5b/config.py

```diff
@@ -0,0 +1,10 @@
+import lsst.meas.algorithms.ingestIndexReferenceTask
+assert type(config)==lsst.meas.algorithms.ingestIndexReferenceTask.DatasetConfig, 'config is of type %s.%s instead of lsst.meas.algorithms.ingestIndexReferenceTask.DatasetConfig' % (type(config).__module__, type(config).__name__)
+# String to pass to the butler to retrieve persisted files.
+config.ref_dataset_name='sdss'
+
+# Depth of the HTM tree to make.  Default is depth=7 which gives
+#               ~ 0.3 sq. deg. per trixel.
+config.indexer['HTM'].depth=7
```

Comment:
Same comment re `default` here as in Jointcal, and in other files in this package.

Reply:
I copied these files from `validation_data_*`, and I no nothing about why these choices were made.

Reply:
That's fine, but can we note that, rather than preserving the confusing comment, please?

Reply:
Does my update to the readme clarify things?